doc/reference/devices_nic.md

```diff
@@ -238,7 +238,7 @@ Key                                  | Type    | Default           | Managed | D
 ### `nictype`: `physical`
 
 ```{note}
-- You can select this NIC type only through the `nictype` option.
+- You can select this NIC type through the `nictype` option or through the `network` option (the provided network needs to be of type `physical`).
 (The [`physical` network type](network-physical) can only be used to provide an uplink network for the {ref}`network-ovn`.)
```

Comment:
```suggestion
- You can select this NIC type through the `nictype` option or the `network` option.
```
The second sentence isn't true anymore now.
For the first one, let's use the same wording as for the other nictypes.